neo/ADoom3.info.src

```diff
@@ -0,0 +1 @@
+TYPE = DRAWER
```

Comment:
same here (neo/sys/aros/)

and the following ADoom3.png I can't comment on because it's no text file - is it even needed here, it looks like you have the normal icons for doom3 and RoE in neo/sys/aros/ already?